chain/network/src/peer_manager/connection_store/mod.rs

```diff
@@ -0,0 +1,114 @@
+use crate::peer_manager::network_state::NetworkState;
+use crate::store;
+use crate::time;
+use crate::types::{ConnectionInfo, PeerType};
+use near_primitives::network::PeerId;
+use parking_lot::Mutex;
+use std::collections::HashMap;
+use std::sync::Arc;
+
+/// The maximum number of stored outbound connections.
+pub const OUTBOUND_CONNECTIONS_LIMIT: usize = 40;
```

Comment:
how about CACHE_SIZE? It is an essentially a LRU cache, so it would be nice to have it explicitly documented (via naming and docstrings).